x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -414,20 +415,21 @@ test('state attribute gets parameterized', async () => {
       },
     ]
   `);
-});
+  });
 
-test(`logs an error when action group isn't part of actionGroups available for the ruleType`, async () => {
-  const executionHandler = createExecutionHandler(createExecutionHandlerParams);
-  const result = await executionHandler({
-    // we have to trick the compiler as this is an invalid type and this test checks whether we
-    // enforce this at runtime as well as compile time
-    actionGroup: 'invalid-group' as 'default' | 'other-group',
-    context: {},
-    state: {},
-    alertId: '2',
+  test(`logs an error when action group isn't part of actionGroups available for the ruleType`, async () => {
+    const executionHandler = createExecutionHandler(createExecutionHandlerParams);
+    const result = await executionHandler({
+      // we have to trick the compiler as this is an invalid type and this test checks whether we
+      // enforce this at runtime as well as compile time
+      actionGroup: 'invalid-group' as 'default' | 'other-group',
+      context: {},
+      state: {},
+      alertId: '2',
+    });
+    expect(result).toEqual([]);
```

Comment:
Should we add `expect` statements to the other tests to ensure that `createExecutionHandler` is now returning non-empty arrays when actions are being enqueued?